src/common/net.cc

```diff
@@ -257,9 +255,7 @@ Address::init(const std::string& addr) {
         }
         try {
             in_addr addr;
-            char buff[INET_ADDRSTRLEN + 1] = {0, };
-            std::copy(host_.begin(), host_.end(), buff);
-            inet_pton(AF_INET, buff, &(addr));
+            inet_pton(AF_INET, host_.c_str(), &(addr));
```

Comment:
Please, merge net.cc with master

Reply:
merged - we now do:
inet_pton(AF_INET6, host_.c_str(), &(addr6.s6_addr16));

i.e: just pass the 'host_' to inet_pton.
I don't know why a copy was done before that (just wasted cycles + valgrind issue)